packages/mdc-dialog/index.ts

```diff
@@ -176,33 +161,40 @@ class MDCDialog extends MDCComponent {
 
   getDefaultFoundation() {
     return new MDCDialogFoundation({
-      addClass: (className) => this.root_.classList.add(className),
-      removeClass: (className) => this.root_.classList.remove(className),
-      hasClass: (className) => this.root_.classList.contains(className),
       addBodyClass: (className) => document.body.classList.add(className),
-      removeBodyClass: (className) => document.body.classList.remove(className),
-      eventTargetMatches: (target, selector) => matches(target, selector),
-      trapFocus: () => this.focusTrap_.activate(),
-      releaseFocus: () => this.focusTrap_.deactivate(),
-      isContentScrollable: () => !!this.content_ && util.isScrollable(/** @type {!Element} */ (this.content_)),
+      addClass: (className) => this.root_.classList.add(className),
       areButtonsStacked: () => util.areTopsMisaligned(this.buttons_),
-      getActionFromEvent: (event) => {
-        const element = closest(event.target, `[${strings.ACTION_ATTRIBUTE}]`);
-        return element && element.getAttribute(strings.ACTION_ATTRIBUTE);
-      },
       clickDefaultButton: () => {
         if (this.defaultButton_) {
           this.defaultButton_.click();
         }
       },
+      eventTargetMatches: (target, selector) => {
+        if (!target) return false;
+        return matches(target as Element, selector);
+      },
+      getActionFromEvent: (event: MouseEvent | KeyboardEvent) => {
+        if (!event.target) return '';
+        const element = closest(event.target as Element, `[${strings.ACTION_ATTRIBUTE}]`);
+        return element && element.getAttribute(strings.ACTION_ATTRIBUTE);
+      },
+      hasClass: (className) => this.root_.classList.contains(className),
+      isContentScrollable: () => !!this.content_ && util.isScrollable(/** @type {!Element} */ (this.content_)),
```

Comment:
Remove `!!`

Reply:
Wouldn't this no longer be boolean without that if `this.content_` doesn't exist? (Though it should really always exist...)

Reply:
Nevermind, I guess this case is handled within `util.isScrollable`.

Reply:
Yeah, I updated `util.isScrollable()` to accept a null `element` argument.

IIRC `content_` can technically be null if you only have a title and action buttons.